src/site/shared/containers/ItemNav/index.tsx

```diff
@@ -14,13 +14,21 @@ import {useHistory} from "shared/utils/hooks/useHistory";
 import {Draggable} from "shared/components/DragDroppable/Draggable";
 import {DragDropHandlers} from "shared/components/DragDroppable/DragDropHandlers";
 
+import {IC} from "digital/models/ioobjects";
+import {ICData} from "digital/models/ioobjects";
+import {DigitalCircuitDesigner} from '../../../../app/digital/models/DigitalCircuitDesigner';
+import {DeleteICDataAction} from '../../../../app/digital/actions/DeleteICDataAction';
```

Comment:
nit: use `"` not `'`

nit: these can be shortened to `from "digital/models/......"`